fdbserver/CoordinatedState.actor.cpp

```diff
@@ -368,3 +368,15 @@ Future<Void> MovableCoordinatedState::setExclusive(Value v) {
 Future<Void> MovableCoordinatedState::move(ClusterConnectionString const& nc) {
 	return MovableCoordinatedStateImpl::move(impl.get(), nc);
 }
+
+Optional<Value> UpdateCCSInMovableValue(ValueRef movableVal, KeyRef oldClusterKey, KeyRef newClusterKey) {
```

Comment:
I think this function name should start with a lower case character to follow the typical naming conventions.

Reply:
oh, nice find! I changed this at the last minute.
Changed.